apps/charterafrica/src/lib/data/common/index.js

```diff
@@ -193,7 +193,30 @@ async function getFeaturedConsultations(consultation, playlistItems) {
   return featured;
 }
 
-async function processPageConsultation(page) {
+async function processDocumentPage(page, api, context) {
+  const { query } = context;
+
+  const { title, ...rest } = query;
+
+  return {
+    ...page,
+    blocks: [
+      {
+        slug: "embedded-document-viewer",
+        options: rest,
+        title,
+      },
+    ],
+  };
+}
+
+async function processPageConsultation(page, api, context) {
+  const { params } = context;
+
+  if (params.slugs.length > 2) {
```

Comment:
Shouldn't we check if it's `opportunities/consultation/documents/<document-slug>`